src/main/java/com/arpnetworking/clusteraggregator/configuration/ClusterAggregatorConfiguration.java

```diff
@@ -429,6 +455,8 @@ public Builder setDatabaseConfigurations(final Map<String, DatabaseConfiguration
         private RebalanceConfiguration _rebalanceConfiguration;
         @NotNull
         private String _clusterHostSuffix = "";
+        @NotNull
+        private Props _clusterJoinActor = new NonJoiningClusterJoiner.Builder().build();
```

Comment:
This isn't actually an actor. Do we want it to be? In the past we've done that through Guice, but as this is configuration I am inclined to say it's Props, but the name should probably reflect that.